server/tests/test_utils.py

```diff
@@ -8,15 +8,56 @@ class MakeMonthDatesTest(unittest.TestCase):
     def test_make_month_dates(self):
         self.assertEqual(
             utils.make_month_dates(1, 2020),
-            (datetime(2020, 1, 1), datetime(2020, 2, 1))
+            (dt(2020, 1, 1), dt(2020, 2, 1))
         )
         self.assertEqual(
             utils.make_month_dates(12, 2020),
-            (datetime(2020, 12, 1), datetime(2021, 1, 1))
+            (dt(2020, 12, 1), dt(2021, 1, 1))
         )
 
 
 class GenerateRandomPasswordTest(unittest.TestCase):
     def test_generate_random_password(self):
         for i in range(0, 100, 10):
             self.assertEqual(len(utils.generate_random_password(i)), i)
+
+
+class MakeMonthDatesRange(unittest.TestCase):
+    def test_month_dates_range(self):
+        self.assertEqual(
+            utils.make_month_dates_range(
+                dt(2020, 1, 1), dt(2020, 1, 31)
+            ),
+            [
+                dt(2020, 1, 1),
+            ]
+        )
+
+        self.assertEqual(
+            utils.make_month_dates_range(
+                dt(2020, 1, 4), dt(2020, 1, 9)
+            ),
+            [
+                dt(2020, 1, 1),
+            ]
+        )
+
+        self.assertEqual(
```

Comment:
Split it into different tests? + No test for multiple years

Reply:
As in separate test for generating ranking and adding match? Or how?